cmd/dashboard/app/dashboard.go

```diff
@@ -44,11 +44,9 @@ func Run(listenAddress string,
 	defaultNamespace string,
 	offline bool,
 	platformConfigurationPath string,
-	templatesGithubOwner string,
-	templatesGithubRepository string,
-	templatesGithubBranch string,
-	templatesGithubAccessToken string) error {
-	var functionGithubTemplateFetcher *functiontemplates.GithubFunctionTemplateFetcher
+	templatesGitRepository string,
+	templatesGitBranch string) error {
```

Comment:
If possible, rather than using Branch use Ref (everywhere). This will allow us to use tags, shas and branches